gateways/clients/elastic_search_client.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Optional
+
+from elasticsearch import Elasticsearch
+
+from common.configuration import ELASTIC_CLOUD_ID, ELASTIC_INDEX, ELASTIC_PASSWORD, ELASTIC_RESULTS_PER_PAGE
+
+
+class ElasticSearchClient:
+    def __init__(self) -> None:
+        if ELASTIC_CLOUD_ID and ELASTIC_PASSWORD:
+            self.client = Elasticsearch(
+                cloud_id=ELASTIC_CLOUD_ID,
+                basic_auth=("elastic", ELASTIC_PASSWORD)
+            )
+
+    def _build_search_query(self, search_text: str, sort_by: str, order: str, search_after: List[str]) -> dict:
+        # Default sort order, if nothing is passed
+        sort_order = ['id', 'name']
```

Comment:
I think we don't have a linter but we've been using single quotes as strings. I think we should try to use only one kind of quotes in the files and this file has both.

Reply:
Thanks for noticing that. I've correct it in a new commit